includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -93,14 +93,7 @@ public static function initialize_runner() {
 
 		foreach ( $runners as $runner ) {
 			if ( $runner->is_plugin_check() ) {
-				add_action(
-					'muplugins_loaded',
-					function() use ( $runner ) {
-						static::$cleanup = $runner->prepare();
-						static::$runner  = $runner;
-					}
-				);
-
+				static::$runner = $runner;
```

Comment:
Can you explain the rationale behind this change? I see you moved the runner `prepare()` call into `Abstract_Check_Runner::run()`, however isn't that too late? The `Universal_Runtime_Preparation` filters the list of active plugins, which has to happen before WordPress loads plugins, which is why here it happens on `muplugins_loaded`.

Getting rid of that will break that behavior I think.

Reply:
@felixarntz I think you're right and this is something I overlooked in the changes here. This should be reverted.

However, I think this highlights that the Runtime_Environment_Setup needs to be run earlier in the process as before the runners prepare method is called as this is what's causing the errors. I'll update this PR and investigate this further here.